src/ksync/test/SConscript

```diff
@@ -29,7 +29,7 @@ env.Prepend(LIBS=['thrift','xmpp', 'peer_sandesh', 'table_sandesh',
 
 if sys.platform != 'darwin':
     if sys.platform.startswith('win'):
-        env.Prepend(LIBS=['windowsstubs'])
+        env.Prepend(LIBS=['Dbghelp', 'windowsstubs'])
```

Comment:
We do not have to include `Dbghelp` here, because it is already included in `MakeTestEnvFn` function.